nbuild.py

```diff
@@ -71,8 +84,9 @@ def main():
     os.environ['PATH'] = '/bin:/sbin/:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/opt/bin'
     os.environ['MAKEFLAGS'] = f'-j{cpu_count() + 1}'
 
-    # Override those with the ones within the .env file.
-    load_dotenv(dotenv_path='.env', override=True)
+    # Override environment with the content of the config file
+    if core.config.get_config().get('env') is not None:
```

Comment:
```suggestion
    if 'env' in core.config.get_config():
```
I think that would be lighter / more idiomatic